common/scripts/create-imodeljs-native-pr-branch.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+usage() {
```

Comment:
Are you going to document which order to run these in? I guess in the internal wiki for now until native is OS?

Reply:
Wiki will direct you to run `upgrade-imodeljs-native.sh`, which invokes the other two. If PR branch creation succeeds but update fails, the upgrade script suggests fixing the problem and running the update script to resume.
I will adjust the upgrade script so that in the case of merge conflicts it recommends fixing the conflicts then continuing by running the update script.

Reply:
Updated the upgrade script for merge conflicts.

Reply:
sounds good.

Reply:
@calebmshafer need anything else here?